readthedocs/projects/models.py

```diff
@@ -1447,7 +1447,6 @@ def __str__(self):
 class WebHook(Notification):
     url = models.URLField(
         max_length=600,
-        blank=True,
```

Comment:
This change will require a migration. `manage.py migrate --name make_url_field_not_blank projects` or similar.

Reply:
Good catch 👍